router.js

```diff
@@ -6,15 +6,22 @@ module.exports = function($window, redraw) {
 	var renderer = createRenderer($window)
 	var router = createRouter($window)
 	var route = function(root, defaultRoute, routes) {
-		var replay = limiter($window, router.defineRoutes(routes, function(component, args) {
-			renderer.render(root, {tag: component, attrs: args})
-		}, function() {
-			router.setPath(defaultRoute)
-		}))
+		var replay = limiter($window, router.defineRoutes(
+			routes,
+			function(component, args) {
+				renderer.render(root, {tag: component, attrs: args})
+			},
+			function() {
+				router.setPath(defaultRoute)
+			}
+		))
 		
 		renderer.setEventCallback(replay)
 		redraw.run = replay
+		
+		replay()
```

Comment:
This is a duplicate of #1048, but I can't test w/o it since nothing will render. If it doesn't merge cleanly I can remove it.